src/libraries/System.Net.HttpListener/tests/HttpListenerTests.cs

```diff
@@ -155,5 +156,30 @@ public void EndGetContext_AlreadyCalled_ThrowsInvalidOperationException()
                 Assert.Throws<InvalidOperationException>(() => listener.EndGetContext(beginGetContextResult));
             }
         }
+
+        [Fact]
+        [OuterLoop]
+        public void GetContext_StopIsCalled_GetContextUnblocked()
+        {
+            using var listenerFactory = new HttpListenerFactory();
+            var listener = listenerFactory.GetListener();
+            listener.Start();
+
+            var listenerTask = Task.Run(() =>
+            {
+                try
+                {
+                    var context = listener.GetContext();
```

Comment:
A reminder on the use of `var`.

Constructs like this:

```c#
var context = listener.GetContext();
```

are not allowed per repo style guidelines because the type of variable is not obvious. Only use 'var' when doing a `new` or a cast.